lib/doc/column.js

```diff
@@ -5,6 +5,8 @@ const _ = require('../utils/under-dash');
 const Enums = require('./enums');
 const colCache = require('../utils/col-cache');
 
+const DEFAULT_COLUMN_WIDTH = 9;
```

Comment:
I'm really happy to read your pull requests, good job. I had some issues with width before, so I feel twice satisfaction with your changes 👍 
but I'm not sure, that it's the correct way to set a const value, 
I think that should be property for worksheet and/or workbook, with a possibility to change it by a user.

May I'm wrong, so please clear me this and tell what are you thinking? 😄

Reply:
Event if I wrote a comment, I think these PR fix some issues anyway, so I approve it to merge 👍

Reply:
@Siemienik First of all, thank you very much for your approval, this will become my motivation. I really like exceljs, it is very helpful for my work, so I am happy to contribute a little bit of my own power to this software.

* When the user does not set the column width, I will set a default value for the column.

```javascript
// single column setting
 sheet.getColumn(3).width = 10;
```

* Users can globally set the column width of the entire worksheet

```javascript
worksheet.properties.defaultColWidth= 47;
```